doc/dbus-interfaces.md

```diff
@@ -76,16 +76,24 @@ Object path: `/org/containers/hirte/monitor/$id`
 
     Stop monitoring and delete the Monitor object, removing any outstanding subscriptions.
 
-  * `Subscribe(in node s, in unit s)`
+  * `Subscribe(in node s, in unit s, out id u)`
 
-    Subscribe to changes in properties of a given unit on a given node, or all nodes if the node name is empty. This
-    will emit the signal `UnitChanged` for all matching units in the system, and then again whenever one of the
-    properties of the unit changes.
+    Subscribe to changes in properties of a given unit on a given node. Passing a wildcard `*` to either `node` or
+    `unit` will subscribe to all nodes or all units on the given node. This will emit the signal `UnitChanged` for all
+    matching units in the system, and then again whenever one of the properties of the unit changes. Returns an
+    identifier `id` used for a subsequent `Unsubscribe`.
```

Comment:
This isn't needed in this PR, but at some point we need to extend this with a fuller description of the events, including the virtual events (and how they differ for wildcarded matches).

Reply:
Good point, I created [an issue](https://github.com/containers/hirte/issues/261) to keep track of this task.